Sources/Actions/Profile/Tracking.php

```diff
@@ -337,9 +337,12 @@ public function activity(): void
 		Db::$db->free_result($request);
 
 		// Find other users that might use the same IP.
-		$ips = array_unique($ips);
+		$ips = array_filter(array_unique($ips), function($ip) {
+			return !empty($ip);
```

Comment:
This was annoying, but the IP can be a empty string, which we then try to use later and things where not happy.